### ISSUE TITLE
Link in plugins package README leads to Github 404 when clicked on in npmjs.com 

### ISSUE THREAD

**Issue by Shelob9** [WordPress/gutenberg#12332]
**Describe the bug**
In the readme for the plugins package, there is the text "See the edit-post module documentation for available components."  The words "edit-post" link to here:

https://github.com/WordPress/gutenberg/blob/edit-post/

As of today November 26th, 2018, that is a 404.

**To Reproduce**
1. Go to https://www.npmjs.com/package/@wordpress/plugins
2. Click on the link to edit-post docs
3. See 404

**Expected behavior**

I'm also confused -- this may belong in a separate issue -- as I assume the link should go here: https://www.npmjs.com/package/@wordpress/edit-post but that says the component is for internal use only, but it appears to be necessary for sidebar plugins according to the plugins readme.

**Comment by designsimply** [comment]
I tested the following:

1. Go to https://github.com/WordPress/gutenberg
1. Enter "See the edit-post module documentation for available components" into the search box at the very very top left and run the search.
1. Click on the `packages/plugins/README.md` link in the results.
1. Find the link text on that page for "the edit-post module documentation" and click on it.

Result: the "the edit-post module documentation" link on the https://github.com/WordPress/gutenberg/blob/3d3c2a4d52903131f68d2694bd551408561fa778/packages/plugins/README.md#L32 page leads me to https://github.com/WordPress/gutenberg/tree/3d3c2a4d52903131f68d2694bd551408561fa778/packages/edit-post

In the source code, the link text "the edit-post module documentation" technically points to `../edit-post/`, so is the problem that the https://www.npmjs.com site simple doesn't know how to handle relative links in documentation like that? Is it possible it's something to address on their side or is there an argument to be made for trying to address the issue on this side even though the links in the files here do seem to be working? Asking because I'm not sure and would like to hear feedback.

**Comment by Shelob9** [comment]
@designsimply You are correct that this happens on the npm page, not from GitHub.

Also, I see here that this is an npmjs bug that should be fixed https://npm.community/t/relative-links-in-npm-readme-markdown-not-functional/1525

**Comment by Shelob9** [comment]
I updated the title to more accurately reflect the issue.  It looks like the solution is to republish the package - https://npm.community/t/relative-links-in-npm-readme-markdown-not-functional/1525/6

**Comment by gziolo** [comment]
We are trying to find a solution in #12835. It's a complex thing to satisfy all 3 parties involved:
- npm
- GitHub
- handbook

If I read properly the link you shared, we might be able to use:
`/packages/edit-post/README.md`

However, this needs some work to be done on `wordpress.org` side :)

**Comment by gziolo** [comment]
It should be fixed once the package is published to npm after #12835 was merged.
